src/test/groovy/org/savantbuild/plugin/release/ReleaseGitPluginTest.groovy

```diff
@@ -139,7 +139,7 @@ class ReleaseGitPluginTest {
     "git add test-pub.txt".execute([], gitDir.toFile()).waitFor()
     "git add test-pub-source.txt".execute([], gitDir.toFile()).waitFor()
     "git commit -am Test".execute([], gitDir.toFile()).waitFor()
-    "git push -u origin master".execute([], gitDir.toFile()).waitFor()
+    "git push -u origin main".execute([], gitDir.toFile()).waitFor()
```

Comment:
Is this correct? I believe `master` is still the default branch in this repo.

Reply:
Yes it is. Because the repo this code is working with is _not_ the savant repos we use it in github. It's a test repo created during this test just run, just for this purpose, with a fake local remote and everything (and that's a good thing). It's just that the test was written in the era before the git CLI's default branch name was `main`.